multimedia/mythtv.27/Portfile

```diff
@@ -65,7 +65,7 @@ if { ${subport} eq ${name} } {
 subport mythtv-core${majorversion} {
 	name                mythtv-core${majorversion}
 	version             0${majorversion}${minorversion}-Fixes-${lastcommit}
-    revision            1
+    revision            2
```

Comment:
This one seems ignored anyway. And spacing is messed up in the whole file.

Reply:
The myth*.27 ports may be dead by now.  I used to have one user that was stuck on that version due to hardware constraints so I left them in place.  I haven't heard from him for a couple of years now so it is probably safe to just remove them.  I'll try to find some time in the near future to do that.

Reply:
I agree this line should just be removed in any case, as it's redundant and potentially confusing.